cmd/swarm/config.go

```diff
@@ -303,6 +308,12 @@ func envVarsOverride(currentConfig *bzzapi.Config) (config *bzzapi.Config) {
 		}
 	}
 
+	if lightnodeenable := os.Getenv(SWARM_ENV_LIGHT_NODE_ENABLE); lightnodeenable != "" {
```

Comment:
Nitpick, but this `var` has a scope of 2 lines, it could very well be just `lne` ;)

Reply:
Changed the name to `lne`